lms/static/sass/course/_gradebook.scss

```diff
@@ -75,7 +75,7 @@ div.gradebook-wrapper {
 		position: relative;
 		float: left;
 		width: 76%;
-		overflow: hidden;
+		overflow-x: scroll;
```

Comment:
#### Overflow
The overflow property specifies what happens if content overflows an element's box.
**Default:** visible - The overflow is not clipped. It renders outside the element's box. This is default
See http://www.w3schools.com/cssref/pr_pos_overflow.asp

#### overflow-x 
The overflow-x property specifies what to do with the left/right edges of the content - if it overflows the element's content area.
**overflow-x: scroll**  The content is clipped and a scrolling mechanism is provided
http://www.w3schools.com/cssref/css3_pr_overflow-x.asp

#### Here:
I am only handling horizontal scroll.

@sarina